tests/Users/create-user.spec.js

```diff
@@ -0,0 +1,129 @@
+import { test, expect } from "@playwright/test";
+
+const getRandomNumber = () => {
+  return Math.floor(Math.random() * (999 - 100 + 1) + 100);
+};
+
+const wait = (seconds) => {
+  return new Promise((resolve) => {
+    setTimeout(resolve, seconds * 1000);
+  });
+};
+
+test.describe("Create user tests", () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto("./");
+    await page.getByPlaceholder("Email Address").fill("root@gmail.com");
+    await page.getByPlaceholder("Password").fill("p@s5W07d");
+    await page.getByRole("button", { name: "Log in" }).click();
+  });
+
+  test.afterEach(async ({ page }) => {
+    // await page.close();
+  });
```

Comment:
Commenting out page.close() might leave browsers open unnecessarily after tests, potentially consuming resources.

Reply:
i commented it out for testing purposes, forgot to uncomment it by mistake